packages/cli/src/databases/repositories/execution.repository.ts

```diff
@@ -467,12 +480,12 @@ export class ExecutionRepository extends Repository<ExecutionEntity> {
 		do {
 			// Delete in batches to avoid "SQLITE_ERROR: Expression tree is too large (maximum depth 1000)" error
 			const batch = executionIds.splice(0, this.deletionBatchSize);
-			await this.softDelete(batch);
+			await this.delete(batch);
```

Comment:
Noticed that we never remove binary data here, even [before the recent refactoring](https://github.com/n8n-io/n8n/pull/7092/files#diff-c8cbb62ca9ab2ae45e5f565cd8c63fff6475809a6241ea0b90acc575615224afL416). Out of scope for this PR, but should do so in future.

Reply:
Good point, created [a ticket](https://linear.app/n8n/issue/PAY-921/should-delete-binary-data-when-when-deleting-multiple-executions-by)